src/os/user/iterate_unix.go

```diff
@@ -0,0 +1,59 @@
+//go:build (aix || darwin || dragonfly || freebsd || (js && wasm) || (!android && linux) || netbsd || openbsd || solaris) && (!cgo || osusergo)
+// +build aix darwin dragonfly freebsd js,wasm !android,linux netbsd openbsd solaris
+// +build !cgo osusergo
+
+package user
+
+import (
+	"os"
+)
+
+func iterateUsers(fn NextUserFunc) error {
+	f, err := os.Open(userFile)
+	if err != nil {
+		return err
+	}
+	defer f.Close()
+	_, err = readColonFile(f, usersIterator(fn), 6)
+	return err
+}
+
+func iterateGroups(fn NextGroupFunc) error {
+	f, err := os.Open(groupFile)
+	if err != nil {
+		return err
+	}
+	defer f.Close()
```

Comment:
not sure what the current convention in the Go stdlib is: are errors from `Close` calls ignored?

Reply:
yes, `lookup_unix.go` ignores deferred `f.Close()` calls